.github/workflows/pm_stale_ticket_labeler.yml

```
@@ -0,0 +1,26 @@
+name: Label inactive issues and pull requests
+on:
+  schedule:
+    - cron: "12 2 * * *"
+
+jobs:
+  label-stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          start-date: 2025-01-01
+          exempt-issue-labels: 'no-automation,"Type: Epic"'
+          exempt-pr-labels: no-automation
+          exempt-assignees: jimchamp,mekarpeles,cdrini,scottbarnes
```

Can we add RayBB, seabelis, and hornc?

```suggestion
          exempt-assignees: jimchamp,mekarpeles,cdrini,scottbarnes,raybb,seabelis,hornc
```